core/src/test/java/org/elasticsearch/search/suggest/AbstractSuggestionBuilderTestCase.java

```diff
@@ -211,6 +213,22 @@ public void testFromXContent() throws IOException {
         }
     }
 
+    protected Tuple<MapperService, SB> mockMapperServiceAndSuggestionBuilder(
```

Comment:
I took a quick stab at sth. that avoids the Tuple and replacing the test builder [here](https://gist.github.com/cbuescher/88531fe7c2abd38936ef), but it still looks a little bit strange to me.